ntbs-service/Models/PatientDetails.cs

```diff
@@ -14,22 +14,26 @@ public class PatientDetails : ModelBase
         [StringLength(35)]
         [RequiredIf(@"ShouldValidateFull", ErrorMessage = ValidationMessages.FamilyNameIsRequired)]
         [RegularExpression(ValidationRegexes.CharacterValidation, ErrorMessage = ValidationMessages.StandardStringFormat)]
+        [Display(Name = "Family name")]
```

Comment:
It looks like Display(Name = "") and DisplayName("") both work for error messages - any preference on which one to use? So far Display(Name = "") is used more widely

Reply:
Glad both work!
I don't think we use any other arguments in `Display()`, so `DisplayName` feels easier on the eyes